example_3_quality_metrics/extraction_to_txt_file.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from speakleash import Speakleash
+
+
+def save_high_quality_docs(txt_folder_name):
```

Comment:
You can split up functionalities using 2 functions:
- `def get_datatxt_folder_name)` which creates directory
- `def save_quality_docs(quality='HIGH')` which will wrap code from creating an instance of the Speakleash class (from line 26 to 41)
- use these two functions inside of condition:` if __name__ == '__main__':`

Look at the following example:
https://github.com/speakleash/speakleash-examples/pull/13/files (file charts.py)